### ISSUE TITLE
Changes pages is empty

### ISSUE THREAD

**Issue by pbrunet** [tobami/codespeed#193]
Hi,

I just downloaded the 10.1 released version and installed it. It almost work the only problem come with the change page which is empty.
Default page contains correct rss information, timeline contains informations too but changes is empty with no possible commit id.

Did you already had this problem? Do you have a solution? (If I add a dummy commit with a commit id which is not a hash, changes appear again).

Thanks,
Pierrick

**Comment by tobami** [comment]
So the changes page does not show an error message? It is just empty?
I am not aware of any issues with changes, but granted the new 0.10.1 release could have a new bug.

Could you elaborate on what kind of data you have? Specially:
1- Do you have more than one project
2- Do you have more than one branch per project?
3- Did you override any settings variables?

That would be very helpful.

**Comment by pbrunet** [comment]
There are no error messages in the Changes page, it is just empty.
My repo is a "None" repo as I didn't provide any access to the git repo. (I
also had a problem with git to import getlogs function but it may be my
fault. I am not sure I correctly set the PYTHONPATH)
I have only one project with on branch (master set in the settings) and
(for now), I have only one environment and one benchmark.
When I click on a "point" in the timeline, I have mean/std/... informations
page.
In the settings page, I just changed "default" to "master" and I set host
information to set my website name. I use the DEBUG mode and the console
logs didn't complain at all.

I don't have the settings here but I can provide it tomorow if you need it.

Thanks for your help.


2015-11-08 11:07 GMT+01:00 Miquel Torres <notifications@github.com>:

**Comment by tobami** [comment]
OK. Just to be clear, the main area in the changes page is empty, but do the navigation bars show an executable you can select?

**Comment by pbrunet** [comment]
Yes, it show an exectuable and an environment but they are unselected by
default and selecting them change nothing. The commitid selected list
contains nothing.

2015-11-09 21:52 GMT+01:00 Miquel Torres <notifications@github.com>:

**Comment by tobami** [comment]
@pbrunet are you still having the same problem?
And a question I forgot to ask, which version of Python are you using?

**Comment by pbrunet** [comment]
@tobami I didn't update codespeed so the problem is still here. I may have a look at this tomorow.

About Python, I use Python 2.7

**Comment by gasparka** [comment]
@tobami
Same problem.
I followed the build guide (build from master), using virtualenv with Python 2.7 (also tested with 3.6) on Ubuntu 16.04, then did the default samples via `` ./manage.py loaddata codespeed/fixtures/testdata.json``. 

Only the comparison page shows data...
![selection_081](https://user-images.githubusercontent.com/16946258/27387138-8a9d6844-56a0-11e7-980e-a017af69542f.png)
![selection_080](https://user-images.githubusercontent.com/16946258/27387248-d51cbfb4-56a0-11e7-8c88-21e0ccac3eea.png)


![selection_082](https://user-images.githubusercontent.com/16946258/27387256-da67fc4a-56a0-11e7-84e0-7055ef0868fd.png)
![selection_083](https://user-images.githubusercontent.com/16946258/27387260-dde95c24-56a0-11e7-9d2a-6da1b5adb66c.png)

I also tried creating new projects and adding samples, but no results show :(

**Comment by KristofferC** [comment]
Exact same problem here. Using the fixtures, the data shows up in "Comparison" but "Timeline" and "Changes" are empty.

**Comment by mwatts15** [comment]
@KristofferC @gasparka I followed similar steps from the latest master. Looking at the logs, it looked like the wrong branch was being queried for reports/results. Setting the DEF_BRANCH in sample_settings/settings.py to "default" so it matched the data, after which I saw reports (on the landing page), as well as on the Timeline and Changes pages.

This is likely different from @pbrunet's issue. I'd be curious to know if they're experiencing problems with the latest revision on master.

**Comment by KristofferC** [comment]
@mwatts15 Yes! Setting the `DEF_BRANCH` made everything show up. Thank you!
